client/src/components/DashboardNav/DashboardNav.tsx

```diff
@@ -0,0 +1,24 @@
+import { Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+
+interface Props {
+  to: string;
+  primary: string;
+}
+
+const DashboardNav = ({ to, primary }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Link to={to} className={classes.link}>
```

Comment:
i would merge Link and Button into one if possible: https://stackoverflow.com/a/59187804/14204824